Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.yml

```
@@ -1301,7 +1301,63 @@ script:
     - contextPath: CarbonBlackEEDR.SearchProcess.results.scriptload_count
       description: The cumulative count of loaded scripts since process tracking started.
       type: Number
-  dockerimage: demisto/python3:3.10.12.63474
+  - arguments:
+    - description: Threat ID to add the tags.
+      name: threat_id
+      required: true
+    - description: The tags to be added to threat.
+      name: tags
+      required: true
+      isArray: true
+    description: Update threat ID tags
+    name: cb-eedr-add-threat-tags
+    outputs:
+    - contextPath: CarbonBlackEEDR.Threat.ThreatID
+      description: Threat ID
+    - contextPath: CarbonBlackEEDR.Threat.Tags
+      description: Output threat ID tags
+  - arguments:
+    - description: Provide Threat ID
+      name: threat_id
+      required: true
+    - description: Provide notes to be added to the provided Threat ID
+      name: notes
+      required: true
+    description: Update threat ID notes
+    name: cb-eedr-add-threat-notes
+    outputs:
+    - contextPath: CarbonBlackEEDR.Threat.ThreatID
+      description: Threat ID
+    - contextPath: CarbonBlackEEDR.Threat.Notes
+      description: Output Threat ID notes
+  - arguments:
+    - description: Provide Alert ID
+      name: alert_id
+      required: true
+    - description: Provide notes to be added to the provided Alert ID
+      name: notes
+      required: true
+    description: Update Alert ID notes
+    name: cb-eedr-add-alert-notes
+    outputs:
+    - contextPath: CarbonBlackEEDR.Alert.AlertID
+      description: Add notes to alert
+    - contextPath: CarbonBlackEEDR.Alert.Notes
+      description: Output alert notes
+  - arguments:
+    - description: Provide tags to search over alerts
+      name: tags
+      required: true
```

```suggestion
      required: true
      isArray: true
```
If possible give some tags